src/App.tsx

```diff
@@ -14,7 +24,9 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie
+          onMovies={handlerAddMovie}
```

Comment:
The prop 'onMovies' is not descriptive. A more appropriate name like 'onAddMovie' would make the code more understandable.